pwsh/buildkarchive.ps1

```diff
@@ -15,16 +18,15 @@ if ($IsWindows) {
 }
 
 if ($IsMacOS) {
-    # Uninstall this because there's only one architecture installed, which
-    # prevents the other architecture of the universal binary from building
+    # We don't need the zstd feature and it will crash at runtime if this one is used anyway
     brew uninstall --ignore-dependencies zstd
 }
 
 $argQt6 = $qtVersion.Major -eq 6 ? '-DBUILD_WITH_QT6=ON' : $null
 $argDeviceArchs = $IsMacOS -and $env:buildArch -eq 'Universal' ? '-DCMAKE_OSX_ARCHITECTURES=x86_64' : $null
 
 # Build
-cmake -G Ninja -DCMAKE_INSTALL_PREFIX="$PWD/installed/" -DCMAKE_BUILD_TYPE=Release $argQt6 $argDeviceArchs -DCMAKE_TOOLCHAIN_FILE="$env:VCPKG_ROOT/scripts/buildsystems/vcpkg.cmake" .
+cmake -G Ninja -DCMAKE_INSTALL_PREFIX="$PWD/installed/" -DCMAKE_BUILD_TYPE=Release $argQt6 $argDeviceArchs -DWITH_BZIP2=OFF -DWITH_LIBLZMA=OFF -DWITH_LIBZSTD=OFF -DCMAKE_TOOLCHAIN_FILE="$env:VCPKG_ROOT/scripts/buildsystems/vcpkg.cmake" .
```

Comment:
`WITH_BZIP2` etc were new CMake flags added in KF6 defaulted to ON, needed to set them OFF to build successfully.